bootstrap-salt.sh

```diff
@@ -3415,9 +3415,22 @@ install_debian_check_services() {
 #
 
 install_fedora_deps() {
+    if [ -n "$_PY_EXE" ] && [ "$_PY_MAJOR_VERSION" -eq 3 ]; then
+        # Packages are named python3-<whatever>
+        PY_PKG_VER=3
+        __PACKAGES="python3-m2crypto python3-PyYAML"
+    else
+        PY_PKG_VER=2
+        __PACKAGES="m2crypto"
+        if [ "$DISTRO_MAJOR_VERSION" -ge 28 ]; then
+          __PACKAGES="${__PACKAGES} python2-pyyaml"
+        else
+          __PACKAGES="${__PACKAGES} PyYAML"
+       fi
+    fi
 
-    __PACKAGES="dnf-utils libyaml m2crypto PyYAML python-crypto python-jinja2"
-    __PACKAGES="${__PACKAGES} python2-msgpack python2-requests python-zmq"
+    __PACKAGES="${__PACKAGES} dnf-utils libyaml python${PY_PKG_VER}-crypto python${PY_PKG_VER}-jinja2"
```

Comment:
After testing this, the Python 3 installation is failing because of the `libyaml` package. @The-Loeki It worked fine when I tested Fedora 28, but when I tested Fedora 27, it failed to find that package. Can you fix that?

Reply:
https://fedora.pkgs.org/27/fedora-x86_64/libyaml-0.1.7-4.fc27.x86_64.rpm.html
http://mirror.nl.leaseweb.net/fedora/linux/releases/27/Everything/x86_64/os/Packages/l/

AFAICT it should 'just' be in the Fedora main repo's?

Reply:
Hrm. It wasn't there when I tested it. If I removed that package, the install went fine.

Reply:
I can test it again, to be sure.

Reply:
Ah, it worked the second time. My apologies. :)